waspc/src/Wasp/Common.hs

```diff
@@ -13,3 +16,12 @@ data DbMigrationsDir
 
 dbMigrationsDirInWaspProjectDir :: Path' (Rel WaspProjectDir) (Dir DbMigrationsDir)
 dbMigrationsDirInWaspProjectDir = [reldir|migrations|]
+
+data StartOptions = StartOptions
```

Comment:
These feel a bit specific to be put here, I would probably put them somewhere under Cli/Command/Start ... .

Reply:
Agreed. Done.